THIRD_PARTY_LICENSE

```diff
@@ -15,7 +15,23 @@ This repository may contain code or content licensed as follows:
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-   
+
+====
+
+   Copyright 2020 Google, Inc. (S2 Geometry)
```

Comment:
Since we aren't using a third_party directory for S2, please add a .gitattributes file in the root with one line.  
```
server/appengine/src/main/java/com/google/** linguist-vendored
```
https://github.com/github/linguist#vendored-code